### DISCUSSION TITLE
epub looks great

### DISCUSSION THREAD

**Discussion by gour** [pprevos/emacs-writing-studio#21] [General]
Hello Peter,

Today I bought your EWS epub via KoboBooks (for my old Kobo reader), but after downloading it on my Debian machine and launching it within Emacs I was stunned how good it looks and quickly uninstalled FBReader which was my favorite epub reader until today.

Thank you very much for creating EWS!!

Thankful and enthusiastic to make most of my writing with the help of Emacs,
Gour

**Comment by pprevos** [comment]
Hi Gour,

Thanks for buying the book.

I am glad that worked out for you.

Feel free to ask questions or point out issues so I can improve on the second edition and paperback version.

P:)
